pkg/controller/perconaxtradbcluster/perconaxtradbcluster_controller.go

```diff
@@ -333,6 +339,60 @@ func (r *ReconcilePerconaXtraDBCluster) deploy(cr *api.PerconaXtraDBCluster) err
 	return nil
 }
 
+func (r *ReconcilePerconaXtraDBCluster) checkSecret(cr *api.PerconaXtraDBCluster, namespace string) error {
+	if cr.Spec.PXC.AllowUnsafeConfig == true {
+		return nil
+	}
+	secretObj := corev1.Secret{}
+	err := r.client.Get(context.TODO(),
+		types.NamespacedName{
+			Namespace: namespace,
+			Name:      cr.Spec.PXC.SSLSecretName,
+		},
+		&secretObj)
```

Comment:
```
...
     &secretObj)
```
-->
```
...
     &secretObj,
)
```

it's a little thing but in the sake of consistency. all other code formatted in that manner